src/integrals.jl

```diff
@@ -0,0 +1,83 @@
+"""
+    SimpsonWeightArray(n::Int; T=Float64)
```

Comment:
I missed that before
```suggestion
    simpson_weight_array(n::Int; T=Float64)
```
Correct the text string of the docs.

Reply:
(my bad.)